### PR TITLE
feat(new_os_versions): add new OS versions to OHIs publish schemas

### PR DESCRIPTION
**Pull request by rubenruizdegauna**
**Versions to be added**
* Ubuntu
  * 20.10 (groovy)
  * 21.04 (Hirsute Hippo)

* Debian
  * 11 (bullseye)

* SUSE 
  * 12.15
  * 15.1
  * 15.2 
  * 15.3

* Oracle Linux (same versions as RHEL)
  * 7
  * 8

### CODE REVIEW

**Review by roobre — COMMENTED** [review]
Looking alright to me from the integrations side!

File: schemas/nrjmx.yml [hunk 1]
```diff
@@ -61,6 +64,9 @@
         - 12.2
         - 12.3
         - 12.4
+        - 12.5
+        - 15.1
+        - 15.2
```
Comment (brushknight) [inline schemas/nrjmx.yml:69]: 15.3?

Comment (cristianciutea) [inline schemas/nrjmx.yml:69]: 15.3?

**Review by cristianciutea — APPROVED** [review]
LGTM

**Pull request merged**